got3/manager/TweetManager.py

```diff
@@ -118,9 +118,9 @@ def getJsonReponse(tweetCriteria, refreshCursor, cookieJar, proxy):
 		]
 
 		if proxy:
-			opener = urllib2.build_opener(urllib2.ProxyHandler({'http': proxy, 'https': proxy}), urllib2.HTTPCookieProcessor(cookieJar))
+			opener = urllib.request.build_opener(urllib.ProxyHandler({'http': proxy, 'https': proxy}), urllib.request.HTTPCookieProcessor(cookieJar))

```

Comment:
Hi, hope you are doing good. 

Could you please verify that your code can resolve all references, as I think ProxyHandler is a class of urllib.request and you are only using urllib.ProxyHandler